webapp/src/hooks/general.test.ts

```diff
@@ -215,3 +219,481 @@ describe('useEnsureProfiles', () => {
         jest.clearAllMocks();
     });
 });
+
+describe('useProfilesInTeam', () => {
+    it('dispatches if no team members have been loaded', async () => {
+        const useSelectorSpy = jest.spyOn(redux, 'useSelector');
+        const profilesInCurrentTeam = [] as String[];
+        const currentTeamId = 'team_id';
+
+        let calls = 0;
+        useSelectorSpy.mockImplementation(() => {
+            if (calls++ % 2 === 0) {
+                return profilesInCurrentTeam;
+            }
+            return currentTeamId;
+        });
+
+        const useDispatchSpy = jest.spyOn(redux, 'useDispatch');
+        const mockDispatchFn = jest.fn();
+        useDispatchSpy.mockReturnValue(mockDispatchFn);
+
+        const {rerender} = renderHook(() => {
+            useProfilesInTeam();
+        });
```

Comment:
Super nit (repeated in the tests below too)
```suggestion
        const {rerender} = renderHook(useProfilesInTeam);
```

Reply:
I think this works for `useProfilesInTeam`, but not `useEnsureProfiles`, since we have to pass a parameter, and then the testing blocks are "different" in ways that might seem more confusing when understanding the whole file.